seccomp.go

```diff
@@ -34,19 +41,28 @@ type VersionError struct {
 	minimum string
 }
 
+// Caches the libseccomp API level
+var apiLevel uint
```

Comment:
I really prefer for the various libseccomp language bindings to be as thin as possible, is there a reason why we need to have `apiLevel` cached in the golang bindings?  Was simply calling `getAPI()` as needed proving to be a problem in your application?

Reply:
I didn't have a problem with simply calling `getAPI()` but I wonder if Cesar had one. His initial commitmsg was:

> In addition, this change caches the libseccomp API level in variable "apiLevel" in order to avoid calling into libseccomp everytime we need to check the API level.

But the C function `seccomp_api_get()` already has a cache, so I don't see a reason to have a second cache in Golang, unless there was a problem with an old version of libseccomp we want to support. I am not aware of such a problem, so we can try to remove this cache.

Reply:
Hi Alban, IIRC, caching apiLevel in Golang was merely an optimization to avoid calling getAPI() all the time. I don't believe there was a functional reason behind it. 

Thanks for driving this PR towards completion!

Reply:
Okay, let's dtop the `apiLevel` caching and just stick with calls to `getAPI()` as needed.

Reply:
Dropped! :)